cspell.json

```diff
@@ -301,6 +304,7 @@
         "inpage",
         "irss",
         "isfacebook",
+        "jkoeaghipilijlahjplgbfiocjhldnap",
```

Comment:
why?

Reply:
check-spell (pull_request) is fail on file : evm-integration 

Goto `chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/dashboard.html#/wallets/transfer` and check the estimated USD value is working.

Reply:
CC @guanbinrui @Jack-Works

Reply:
I think it's OK to ignore the extension id.
Or we can use inline ignore, like

```md
<!-- cspell:ignore jkoeaghipilijlahjplgbfiocjhldnap -->
```